tests/files.rs

```diff
@@ -120,28 +99,28 @@ fn generate_tests(glob: &str) -> Vec<Trial> {
     for entry in glob::glob(glob).unwrap() {
         let run = Run {
             path: entry.unwrap().clone(),
-            test_proofs: false,
-            resugar: false,
-            test_terms_encoding: false,
+            // test_proofs: false,
+            // resugar: false,
```

Comment:
We should make re-sugaring work and make sure to test it- it's been very valuable in the past for catching bugs

Reply:
It works now!